rapids-4-spark-tools/src/main/scala/com/nvidia/spark/rapids/tool/profiling/QualificationMain.scala

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids.tool.profiling
+
+import java.io.FileWriter
+
+import scala.collection.mutable.ArrayBuffer
+
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.fs.{FileSystem, Path}
+
+import org.apache.spark.internal.Logging
+import org.apache.spark.sql.{DataFrame, SparkSession}
+import org.apache.spark.sql.rapids.tool.profiling._
+
+/**
+ * A tool to analyze Spark event logs and determine if 
+ * they might be a good fit for running on the GPU.
+ */
+object QualificationMain extends Logging {
+  /**
+   * Entry point from spark-submit running this as the driver.
+   */
+  def main(args: Array[String]) {
+    val sparkSession = ProfileUtils.createSparkSession
+    val (exitCode, optDf) = mainInternal(sparkSession, new ProfileArgs(args))
```

Comment:
I think we need separate `QualificationArgs` instead of showing `ProfileArgs` so that we can show appropriate information when running `--help`.